client/analytics/filters/index.js

```diff
@@ -0,0 +1,84 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { noop } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import AdvancedFilters from 'components/advanced-filters';
+import Card from 'components/card';
+import DatePicker from 'components/date-picker';
+import FilterPicker from 'components/filter-picker';
+import { H, Section } from 'layout/section';
+import './style.scss';
+
+const ReportFilters = ( { advancedConfig, filters, filterPaths, query, path } ) => {
+	let advancedCard = false;
+	switch ( query.filter ) {
+		case 'compare':
+			advancedCard = (
+				<Card
+					title={ __( 'Compare Products', 'wc-admin' ) }
+					className="woocommerce-analytics__compare"
+				>
+					<div className="woocommerce-analytics__compare-body">
+						<input type="search" />
+						<div>Tokens</div>
+					</div>
+					<div className="woocommerce-analytics__compare-footer">
+						<Button onClick={ noop } isDefault>
+							{ __( 'Compare', 'wc-admin' ) }
+						</Button>
+					</div>
+				</Card>
+			);
+			break;
+		case 'advanced':
+			advancedCard = (
+				<AdvancedFilters config={ advancedConfig } filterTitle={ __( 'Orders', 'wc-admin' ) } />
+			);
+			break;
+	}
+
+	return (
+		<Fragment>
+			<H className="screen-reader-text">{ __( 'Filters', 'wc-admin' ) }</H>
+			<Section component="div" className="woocommerce-analytics__filters">
+				<div className="woocommerce-analytics__basic-filters">
+					<DatePicker query={ query } path={ path } key={ JSON.stringify( query ) } />
+					{ !! filters.length && (
+						<FilterPicker
+							query={ query }
+							path={ path }
```

Comment:
Do we need to pass through `path`/`query`? Since this should be based on the current page, can each picker [access these from `history.location`](https://github.com/woocommerce/wc-admin/blob/master/client/lib/nav-utils.js#L23-L24) (maybe via helper function in `lib/nav-utils`)?